### ISSUE TITLE
Targetting java 1.8 is not that great

### ISSUE THREAD

**Issue by maxleiko** [kevoree-modeling/framework#24]
By compiling KMF for Java 1.8, you say goodbye to the Android-world unfortunately. It looks like you cannot convert Java-code in 1.8 to a dex file :(

I've just tried to compile the framework with 1.7, and it works. Maybe it's worth compiling for 1.7 ?

**Comment by dukeboard** [comment]
Yep we compile in Java 1.8 just to have enhanced bytecode with closure, However the framework compile fine with Java 1.6 and of course Android as well
